ome_model/experimental.py

```diff
@@ -230,12 +225,12 @@ def create_companion(plates=[], images=[], out=None):
 
     # https://stackoverflow.com/a/48671499/56887
     kwargs = dict(encoding="UTF-8")
-    if PYTHON >= 3:
-        kwargs["xml_declaration"] = True
-        if not out:
+    kwargs["xml_declaration"] = True
+    if not out:
+        try:
             out = sys.stdout.buffer
-    elif not out:
-        out = sys.stdout
+        except AttributeError:
```

Comment:
Happy with either the `try/catch` approach or the `getattr` one-liner.

Is it possible to add a comment similar to https://github.com/Yelp/mrjob/pull/1443 explaining the special handling? Is there a reference page explaining why the stdout is different in the Jupyter notebook environment?

Reply:
Updated. I'm not aware of an explanation, my best guess is it was originally written to match python2 `sys.stdout` and no one though of adding `.buffer` to it.